code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -0,0 +1,175 @@
+	result_atoms = list(/obj/structure/eldritch_crucible)
+
+/datum/eldritch_knowledge/void_blade_upgrade
+	name = "Seeking blade"
+	gain_text = "Fleeting memories, fleeting feet. Not a single human can escape me now."
+	desc = "You can now use your blade on a distant marked target to move to them and attack them."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/spell/voidpull)
+	banned_knowledge = list(/datum/eldritch_knowledge/ash_blade_upgrade,/datum/eldritch_knowledge/flesh_blade_upgrade,/datum/eldritch_knowledge/rust_blade_upgrade)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/void_blade_upgrade/on_distant_eldritch_blade(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!ishuman(target) || !iscarbon(user))
+		return
+	var/mob/living/carbon/carbon_human = user
+	var/mob/living/carbon/human/human_target = target
+	var/datum/status_effect/eldritch/effect = human_target.has_status_effect(/datum/status_effect/eldritch/rust) || human_target.has_status_effect(/datum/status_effect/eldritch/ash) || human_target.has_status_effect(/datum/status_effect/eldritch/flesh) || human_target.has_status_effect(/datum/status_effect/eldritch/void)
+	if(!effect)
+		return
+	var/dir = angle2dir(dir2angle(get_dir(user,human_target))+180)
+	carbon_human.forceMove(get_step(human_target,dir))
+	var/obj/item/melee/sickly_blade/blade = carbon_human.get_active_held_item()
+	blade.melee_attack_chain(carbon_human,human_target)
+
+/datum/eldritch_knowledge/spell/voidpull
+	name = "Void Pull"
+	gain_text = "Aristocrat gave me a favor, i won't waste it..."
+	desc = "You gain an ability that let's you pull people around you closer to you."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/void_pull
+	next_knowledge = list(/datum/eldritch_knowledge/final/void_final,/datum/eldritch_knowledge/spell/blood_siphon,/datum/eldritch_knowledge/summon/rusty)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/final/void_final
+	name = "Call of the Void"
+	desc = "Bring 3 corpses onto the transmutation rune. After you finish the ritual you will automatically silence people around you aswell as passively cool the environment."
+	gain_text = "Calm...... Absolute..... ETERNAL...."
+	cost = 3
+	required_atoms = list(/mob/living/carbon/human)
+	route = PATH_VOID
+
+/datum/eldritch_knowledge/final/void_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	var/mob/living/carbon/human/H = user
+	H.physiology.brute_mod *= 0.5
+	H.physiology.burn_mod *= 0.5
+	H.client?.give_award(/datum/award/achievement/misc/void_ascension, H)
+	priority_announce("$^@&#*$^@(#&$(@&#^$&#^@#  SILENCE! the Void Seer [user.real_name] has come! Fear his call! $^@&#*$^@(#&$(@&#^$&#^@#","#$^@&#*$^@(#&$(@&#^$&#^@#", 'sound/ai/spanomalies.ogg')
+	return ..()
+
+/datum/eldritch_knowledge/final/void_final/on_life(mob/user)
+	. = ..()
+	if(!finished)
+		return
+
+	for(var/mob/living/carbon/livies in spiral_range(7,user)-user)
+		livies.silent += 5
```

Comment:
this will affect your minions

also, this looks to be uncapped, so if you stand around someone long enough, you could mute them for, like, dozens of minutes, even after you walk away from them.

Reply:
fixed